README.md

```diff
@@ -72,7 +83,25 @@ Belle is friends with Mrs. Potts
 
 Write your code here:
 ```ruby
-# code here
+class Friends
+  attr_accessor :friends
+
+  def initialize = friends
+    @friends =friends
+    friends = ["Chip Potts", "Cogsworth", "Lumière", "Mrs. Potts"]
+  end
+
+  def greeting friends
+    @friends.each do |friend|
+    puts "Belle is friends with #{@friends}"
+  end
+
+end
+
+chip = Friends.new "Chip Potts"
+cogs = Friends.new "Cogsworth"
+lumi = Friends.new "Lumière"
+mrs = Friends.new "Mrs. Potts"
 ```
```

Comment:
This example does not require you using a class, in fact I think it might make it a little more difficult,
since we already have the `friends` array you can just use you `.each` loop to get the desired output:

```ruby
friends.each do |friend|
 puts "Belle is freinds with #{friend}"
end
```